src/css/components/_control.scss

```diff
@@ -9,8 +9,14 @@
   height: 100%;
   width: 4em;
   @include flex(none);
+}
 
+.video-js .vjs-control.vjs-visible-text {
+  width: unset;
```

Comment:
`unset` is not supported in IE11. Have you tried this PR there?

Reply:
I guess ie is still somehow a thing :/ Updated.